nss-day-cohort-44/rare-rest-api-malicous-vortices
# Kn post edit server

# ISSUE TICKET 
## Description
update function for the posts is working along with the list checking if a post was made by the current signed in user if so then it makes a property true for filtering the display of the edit button on the client side

## Type of change

# Testing Instructions
`git fetch origin `
`git checkout ` 

User Instructions: 
pull down the client side pull request `Kn-post-edit` along with this pull request run both and go to the client application and test out the edit button on view all posts and you should be able to edit a post.
# Checklist:
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] My changes generate no new errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
